policyengine_us/parameters/gov/states/tx/tanf/one_time_tanf/one_time_monthly_income_limit.yaml

```diff
@@ -13,19 +13,24 @@ brackets:
       2023-01-01: 1
```

Comment:
Is there a reference for the previous values? Want to make sure that the value did not update in 2024 as well